### PR TITLE
feat: PNPM

### PR DESCRIPTION
**Pull request by kristianbinau**
## Description

Change to PNPM

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [x] 🧹 Code refactor
- [x] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by kristianbinau** [comment]
Everything seems to work except the test that's just waiting. I think GH is confused...

File: Makefile [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
+	pnpm run start
 dev:
-	npm run dev
+	pnpm run dev
 test:
-	npm run test
+	pnpm run test
 test-coverage:
-	npm run test -- --coverage
+	pnpm run test -- --coverage
```
Comment (frederikpyt) [inline Makefile:17]: In the test action you changed it to:
```suggestion
	pnpm run test --coverage
```

File: Makefile [hunk 2]
```diff
@@ -1,19 +1,19 @@
 migrate-dev:
-	npx prisma migrate dev
+	pnpm exec  prisma migrate dev
 migrate-deploy:
-	npx prisma migrate deploy
+	pnpm exec  prisma migrate deploy
 migrate-reset:
-	npx prisma migrate reset
+	pnpm exec  prisma migrate reset
```
Comment (frederikpyt) [inline Makefile:6]: Double space

**Review by frederikpyt — CHANGES_REQUESTED** [review]

File: Makefile [hunk 3]
```diff
@@ -1,19 +1,19 @@
 migrate-dev:
-	npx prisma migrate dev
+	pnpm exec  prisma migrate dev
```
Comment (kristianbinau) [inline Makefile:2]:
```suggestion
	pnpm exec prisma migrate dev
```

File: Makefile [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
 migrate-deploy:
-	npx prisma migrate deploy
+	pnpm exec  prisma migrate deploy
```
Comment (kristianbinau) [inline Makefile:4]:
```suggestion
	pnpm exec prisma migrate deploy
```

File: Makefile (hunk 2 above)
Comment (kristianbinau) [inline Makefile:6]:
```suggestion
	pnpm exec prisma migrate reset
```

**Review by frederikpyt — APPROVED** [review]

**Pull request merged**